lib/rules/next-tick-style.js

```diff
@@ -123,6 +150,66 @@ module.exports = {
           return
         }
 
+        if (preferredStyle === 'await') {
+          if (
+            callExpression.arguments.length > 0 ||
+            callExpression.parent.type !== 'AwaitExpression'
+          ) {
+            context.report({
+              node,
+              messageId: 'useAwait',
+              fix(fixer) {
+                const sourceCode = context.getSourceCode()
+
+                // Handle callback to await conversion
+                if (callExpression.arguments.length === 1) {
+                  if (callExpression.parent.type !== 'ExpressionStatement') {
+                    return null
+                  }
+
+                  const [args] = callExpression.arguments
+                  let callbackBody = null
+
+                  callbackBody =
+                    args.type === 'ArrowFunctionExpression' ||
+                    (args.type === 'FunctionExpression' && !args.generator)
+                      ? extractCallbackBody(args, sourceCode)
+                      : `${sourceCode.getText(args)}()`
```

Comment:
This is also not safe

```js
nextTick(foo || bar)
```

```diff
- await nextTick(); foo || bar()
+ await nextTick(); (foo || bar)()
```

If `foo` is truly, it will not be called